terraform/gcp/modules/fulcio/security_policy.tf

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2022 The Sigstore Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Enable required services for this module
+resource "google_compute_security_policy" "fulcio" {
+  name    = "fulcio-service-security-policy"
+  project = var.project_id
+  type    = "CLOUD_ARMOR"
+
```

Comment:
Can we add a content length restriction here too? Someone could construct a massive CSR